cnf-app-mac-operator/config/manifests/bases/cnf-app-mac-operator.clusterserviceversion.yaml

```diff
@@ -19,8 +19,8 @@ spec:
oPkGMAybWqnN3DlWdX6T95wCwooGUEXoInHWfrN4pdwobqztJE8SL73nrz9hFPniQFZj2qu8ijCXj/WrfQaFUjG4UdqJXn3C+QbqsqllRm36yDpe3tS9jYCwnpqrCFtt39CXias3l16oy7ztpNegUIJBDo1vH1M1OEr1cyhK5WhIO1jPg8KsVJRavRjnc5WrQyUEvlNCqWAAnNZFe/kMQo0DwJ231JZr3PXF+ycBYQ0DqCJc6WCE+tpARpTPLwSF612M1JcR5HyzeL8Eg0tLf+nJtkolYeO7JZ5gKFWEw66U1rMmh7fyc7B72p+Fcn53yz93q5znr5TqkAnIMGyb3KhsSIMu1GXeUnvjrPcNpT128Rr3sFzjXl2cK/PU0j/JaV0vN4e12muzpJlHlFNKe+UHaAjOl/ead2C+V+YFfPBYdSfLcmO5U7Y+323xJjGHhLdTSi/VuIHOAM/5US6VxwefIaUUJfQcXlHknJt/mIRNzhnKexbl3ZRrCgo1D1Jtwv01B/36sFUGwK6Ua9md8n4Xl9C9Xf7cOTTvZf7/d05b/RYQ1pSrCCmlmxWvd/5iToil2tGKpmnm5adNrQKxrovl8XI8uklcvEE86vhcKI++bhpzxeNcZReNub2yvO/QzvlJtiq+AADd2vR+BHnZ64ebHhQq1/LrI/tub8OQ9qu4dMS1rLUBPQHhbK5XviHS9Q5GB65WXjk5jcUbxFpuFPNFY6dpmqrW9y9VhGtlCU+AoathP4I3S5vItVUr2mdRquFXRzjo0yb7VRTmIJzBACbvXmp7wm6pSLy19AJtqHKvhKZpbjnnwBg0TXO3krlVV/pYfa3Mqbpa9mp5p4wwCwfHs+N1ISCcXc0rGkVH7+/qQCevDkGteyUMZZI6wJMc1Vq6CYvLNH+w+tpZv/tLIMj/nr2FVdfetFfL6VQUIDdOi9EZ5SpC5XMRzrc9Kaq0nVweQatRreZ7JVwuVaqNKz/nV7UaASNwq7Ib5oPJp/HoJv9BGcmeB5nDE1HjUPvw/H+bW9WOfNwnH6gEhHZUPxchj0602etYJmm/NvAVbmo23ytht4xobFxuNao8DAOcxq0yql6r8+Xhhn8zJh8QQotROwYwF2GrtIi0Kq9qVPl+EENX414J+efo3tKzAANRBsvMq+I4OUBOnoDQntp7tF/tYtOwstfCg6Vnact8r4QqVjcqF1ZzUIChq27vGepQBn0nPxAmILSk3DjVvkti6xOWy+e+7Iaxc2+W5UY3rrQ8VbUcK8AqyupsBrc4zuQDpIDQrr3Kb5TzMpo7S8+eUblh3BUSOpf3SqjiS6tcXF9aegFgOGpfhZDNuTH1exoBoUUDqSJ08v5KSKhihHvkcsi7VcMyqGUn0DeWXgAYgPIdporAknI/N+m5CAJC+2qvIlzqatJr+bI1qty9F2vZK6FpmmuVT9AHOIl2SY4z6QqTgNCygVQROmtTERJ6kyect94uto6maa4KCcAQ5SW7rWjEUcpk5deOeGkSBIRu1F5FON/lqjglJHxy6v17HcqrK+yWC1sVSkjQbgQM0VWtRhyj9vu5zggIHShVhNpLU9e7bFFZmLjsS7ddb5VwUMXmaYtKu5HqETAoC6vxwWMWlvaeHAGhI03T7FV+c3y+60nF5SZ2x3rCrXmlaZrLbe6I3bZSPfqM6hEwJOV6ZYCDJWXVvslVyAWEbtVeRbjW9UTXfDPbNM2O9pMzyUHzkyV0Vq98me4KhsCQlAEO86lYUirkk7qmCQgdGsASalt9Taguv1xGlleXg9VOjS1FJ1loMXOxBQbDogucYFIDXwJC92qvIuR19S8sPduBMrKc/1t3+v6QA5SD5Qs5WNXcUnSSUj26KhgCQyIkcJRyLZ5MSBAQOjaQjVh6a10pN427bhpP9FqpGlSzStFZLATDKS0luF9+xoEBKiHBnAQeM6WQICD0o/YqwotdbZ52nIWbRnMTvi9XVp5vmub6UKsGxynB8HK5aR77ylY5CF0oP+PAQJUBvhcmMpj1wKqDp7MQEkZdZRIQelC+ZGpvq+k9xJSbxjw34fmJtx3dKe1Eu2VjltHKN81N01woVZKxXXTnbWFVrzQFnF6p5I69AvpGWXFw1NefNi200L401gApIPSn9irCpS43TztJvikubUcvTCwo5M/6UgkGo2gnOq1cJSkX3TEEhfz+X8vBZ2rnEabgUAV0TDeD86r1YOe6bVoZAN4ZY4AUEHpSbhxUEU6Qj9FCUBhz6W6xYnBj6dWJKBfdxaAwtPL2fnnfF8rnAEZsoTV26AMbk6la96EMcl4e2yCngNCv2m8izqeUOt087TRKULhaWo+GeON4lP1Sxn1+ihWDk8yDQmk9emkAIzH3SuXn3BjniwDHWxjYmO/vM6SgcHMhGLgGtezQIOfgKwoCQo+GUkXoevO00yqpfH7j+Jny5TakL+P98p4/U24mrxmtOVmuqJSRmBwOX6lopYgH5WYgb1i3M+XKD3BwfbpWKgqvVDyQ9aAMtOXBqauCQfdKUKjxOnacB+VeJQ/QHVzbUtM0kVLKafjVY/7BjWiaJtX0ftpSVgt6u/K3+VrNLRMppctlBYH85/mlv7BZ+YsgfwHftopNO0pgnZ/z3R7P+fxc3lh3o7qUUrP05AaM5fvUtWrzUkr5d+JSRW/pThm13biUUq4qXK3g2rTWd1ct53aMv1Nlv6nF69jW0l/qx4MyGT2f6/yzcfe4gcsPAgIMVfml2y3l3vznxR4/yn75BZv/ot3WbtK9cs53Fs75hRYuxg/mX5bz8+lcAutauDbNr09dXpvuLHx3uQ4NQAmTO+X61dZ1bG5+b/Kw/JkDwP1Vq0cCAqNTfvEuLPzyzXeKXmdkZN4S9vDQL5wbyMqUC/L8XOdzf1Kr3PyL8v5xoycAbSodBPPvqXPle2pu54hR5XvlmjN3vzwObvx8f41LqZgv/lw8qTp2d+FnpNX7EgEBAAB4JCL+fwRBcOn9x9KJAAAAAElFTkSuQmCC
```

Comment:
Please consider downloading a smaller icon from https://www.redhat.com/en/about/brand/standards/icons/standard-icons (Download icons (Red Hat credentials required)) There you will find a small favico that is only 4KB

Reply:
Ack, I'll update all images then, thanks